serving/installing-logging-metrics-traces.md

```diff
@@ -1,29 +1,17 @@
 # Monitoring, Logging and Tracing Installation
 
-Knative Serving offers two different monitoring setups:
-[Elasticsearch, Kibana, Prometheus and Grafana](#elasticsearch-kibana-prometheus--grafana-setup)
-or
-[Stackdriver, Prometheus and Grafana](#stackdriver-prometheus--grafana-setup)
-You can install only one of these two setups and side-by-side installation of
-these two are not supported.
+Knative Serving offers three different monitoring setups:
+[Elasticsearch and Kibana for logging, Prometheus and Grafana for metrics Setup](#elasticsearch-and-kibana-for-logging--prometheus-and-grafana-for-metrics-setup), [Stackdriver for logging, Prometheus & Grafana for metrics Setup](#stackdriver-for-logging--prometheus--grafana-for-metrics-setup), or [Stackdriver for both logging and metrics](#stackdriver-for-both-logging-and-metrics).
```

Comment:
Im trying to reauthor all my edits in #447 to accommodate all the changes in 0.2.0 (including the updates here) and have a few questions.

1. Is it possible to achieve these installation options by using the `monitoring-*.yaml` files that we have listed on the [serving release page](https://github.com/knative/serving/releases)? Would this instead the recommended method (using the listed install packages)?

2. Ive tested `monitoring.yaml` file and found that it installed the ELK stack. Is there an install package for Stackdriver? 

3. If you install `monitoring.yaml`, is it correct that all you need to run is following to get "Stackdriver observability plugin"? 
`kubectl delete       -f third_party/config/monitoring/logging/elasticsearch \
     -f config/monitoring/logging/elasticsearch

Reply:
Moving questions about using *.yaml files to #613